lib/http2/hpack.c

```diff
@@ -669,17 +669,15 @@ size_t h2o_hpack_encode_string(uint8_t *dst, const char *s, size_t len)
     return encode_as_is(dst, s, len);
 }
 
-static uint8_t *encode_header(h2o_hpack_header_table_t *header_table, uint8_t *dst, const h2o_iovec_t *name,
-                              const h2o_iovec_t *value)
+static uint8_t *do_encode_header(h2o_hpack_header_table_t *header_table, uint8_t *dst, const h2o_iovec_t *name,
+                              const h2o_iovec_t *value, int name_index, h2o_header_flags_t flags)
```

Comment:
Is there a need to accept `name_index` as a separate argument? I assume the value is available as `flags.http2_static_table_name_index`.

Reply:
oh that's my mistake